src/components/Card/index.js

```diff
@@ -3,9 +3,12 @@ import {Heading, Paragraph} from 'egghead-ui'
 
 export default ({children, title, description}) => (
   <section className='mb4'>
-    <Heading level='4'>
-      {title}
-    </Heading>
+    {title
+      ? <Heading level='4'>
+          {title}
+        </Heading>
+      : null
+    }
```

Comment:
Fixes the prop type error introduced in https://github.com/eggheadio/egghead-instructor-center/pull/166. Not sure I like the title being moved down like this as it is being applied to the video tile in an inconsistent way than the other headings, but this at least fixes the prop type error.